### ISSUE TITLE
为什么你们的代码中，每条数据是随机分配给随机的老师模型，并不是每个数据根据任务去分配给对应的老师模型？

### ISSUE THREAD

**Issue by shoutOutYangJie** [worldbench/DanceOPD#2]

**Comment by weichow23** [comment]
Hi!
Due to ByteDance Seed’s release restrictions, we are unable to publish the checkpoints and the original training code. The code in this repository is an external reimplementation, which unfortunately contained some mistakes. The hard route is indeed very important, and our previous implementation was incorrect.
We have now thoroughly reviewed the correspondence between the code and the paper and updated the paper accordingly. Thank you again for your careful review and support! Please let us know if you notice any other issues. 😊
